tests/subsys/dfu/img_util/prj_native_posix.conf

```diff
@@ -1,5 +1,4 @@
 CONFIG_ZTEST=y
-CONFIG_STDOUT_CONSOLE=y
```

Comment:
This seems quite unrelated, or?

Reply:
I agree, I just tried to be sneaky and fix an annoying warning, I will remove it.

Reply:
Don't be sneaky ;), that will confuse people looking at the history. Just send a separate commit and/or separate PR